src/Bowling.js

```diff
@@ -0,0 +1,54 @@
+function Scorecard() {
+  this.score = 0;
+  this.frames = [];
+  this.currentFrame = [];
+  this.frameCounter = 0;
+};
+
+Scorecard.prototype.rolls = function (roll1, roll2) {
+  this.currentFrame = []
+  this.currentFrame.push(roll1, roll2)
+  this.score = this.score + (this.currentFrame.reduce((a, b) => a + b, 0));
```

Comment:
could you extract the reduce method to maintain single responsibility principle?